flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -1041,7 +1045,13 @@ public void restore(Collection<KeyedStateHandle> restoreState) throws Exception
 
 	@Override
 	public void notifyCheckpointComplete(long completedCheckpointId) {
+
+		if (!enableIncrementalCheckpointing) {
```

Comment:
👍

Reply:
I think it would actually be easier to understand when written:
```
if (incrementalCheckpointingEnabled) {
  // do clean up work for local sst files which is only relevant for incremental checkpointing
}
```